src/internal/Subscription.ts

```diff
@@ -170,14 +169,14 @@ export class Subscription implements SubscriptionLike {
 
     // Add `this` as parent of `subscription` if that's not already the case.
     let { _parentOrParents } = subscription;
-    if (_parentOrParents === null) {
+    if (_parentOrParents == null) {
```

Comment:
thought: Yikes. I wonder if there's a lint rule to catch this sort of thing?

Reply:
This is intentional.  `== null` matches `null` or `undefined`.

Reply:
Yeah. I know, your change fixes a potential bug. I'm wondering if comparisons like this could be elsewhere. I know there is a lint rule that relaxes `===` when comparing `null`, but IDK that there's one that enforces/recommends the use of `==` when comparing against `null`. With the varied use of `null` and `undefined` in this codebase - callers can pass either - it would be interesting - to me, anyway - to see if this occurs elsewhere. A search could find it, but a lint rule could enforce it.

Reply:
It seems that there are about half a dozen other uses of either `=== null` or `=== undefined` throughout the codebase (ignoring the docs app).